.github/workflows/build-and-publish.yml

```diff
@@ -2,10 +2,12 @@ name: Build and Publish
 
 on:
   push:
+    branches: [ 'main' ]  # Trigger on push to 'main'
+    tags: [ '[0-9]+.[0-9]+.[0-9]+' ]  # Trigger on version tags like '0.1.0'
```

Comment:
Do we want to trigger on tags? My understanding is we tag after 'main' (or 'master' in older repos) has already been released.

Reply:
Based on [this comment](https://github.com/silinternational/postgresql-backup-restore/pull/13/files/b0d6e2ee1f37baf00ea013e9c93ed77778bf52f5#diff-c58efb7d239ba7eb89123ff4b9e3117bd84c382787283c3f149cf0c4620d7029) from @briskt, I think we do trigger on tags now, though I am not up-to-speed on all of the details.

Reply:
That's correct. Previous to this PR, we triggered on `push` with no qualifiers, which meant all branches and all tags.